api/app/models.py

```diff
@@ -0,0 +1,39 @@
+from app import db
+from sqlalchemy.dialects.postgresql import JSON, ARRAY
+
+class election(db.Model):
+    __tablename__ = 'election'
+    id = db.Column(db.Integer, primary_key = True)
+    name = db.Column(db.String)
+    description = db.Column(db.String)
+
+    questions = db.Column(ARRAY(db.Integer))
+
+    hasVoted = db.Column(JSON) # ids of users who have voted in this election w/ timestamp
+
+    active = db.Column(db.Boolean)
+    creator = db.Column(db.Integer)
+    deadline = db.Column(db.DateTime)
+
+class question(db.Model):
+    __tablename__ = 'question'
+
+    id = db.Column(db.Integer, primary_key = True)
+    question = db.Column(db.String)
+    votes = db.Column(JSON) # Options and results
+    voteType = db.Column(db.String) # Proportional vs SPTP vs Ranked
+
+
+class user(db.Model):
+    __tablename__ = 'user'
+
+    id = db.Column(db.Integer, primary_key = True)
+    userName = db.Column(db.String)
+
+    uuid = db.Column(db.Integer)
+
+    canVote = db.Column(ARRAY(db.Integer)) # id numbers of whitelisted elections
+
+    boardMember = db.Column(db.Boolean)
+    email = db.Column(db.String)
```

Comment:
The email is redundant data; we can extract a user's email from the Membership Portal API; you can extract that data from there.

Come to think of it, we'll need a way to initialize the database by automatically adding all the members from the portal to our own; the `GET api/v1/leaderboard` route might help here. That can be done after the routes, though.